tests/test_auth_basic_auth.py

```diff
@@ -95,3 +95,4 @@ def test_expired_cookie(self):
         self.app.set_cookie(settings.COOKIE_NAME, str(cookie))
         res = self.app.get(self.reachable_url)
         assert_equal(res.status_code, 302)
+        assert_in('login', res.location)
```

Comment:
This is fine. To be even more specific, I believe you can do a direct comparison against `res.request.path`. But it's no big deal.